packages/ui-extensions/src/surfaces/admin/components.d.ts

```diff
@@ -1,9 +1,13 @@
-/** VERSION: 0.28.0 **/
+/** VERSION: 0.29.0 **/
 /* eslint-disable @typescript-eslint/ban-types */
 /* eslint-disable @typescript-eslint/no-namespace */
 /* eslint-disable @typescript-eslint/member-ordering */
 // eslint-disable-next-line @typescript-eslint/triple-slash-reference, spaced-comment
 /// <reference lib="DOM" />
+import {ReactNode} from 'react';
+
+import * as preact from 'preact';
```

Comment:
ran this using the latest in https://github.com/Shopify/admin-ui-foundations/pull/1918 
@vividviolet is importing from these an libraries expected result?